src/lib/actions.js

```diff
@@ -8,17 +8,19 @@ import cookies from 'js-cookie';
 
 export const clearSignedInState = store.action(() => {
   const {isSignedIn} = store.getState();
-  if (isSignedIn) {
-    return {
-      userUrlSeen: null,
-      userUrl: null,
-      checkingSignedInState: false,
-      isSignedIn: false,
-      user: null,
-      lighthouseResult: null,
-      lighthouseError: null,
-    };
+  if (!isSignedIn) {
```

Comment:
If possible I'd like to use the noImplicitReturns option and switch this code back to the previous version.

It might be personal opinion, but I think early returns that check for truthiness are often easier to grok:

`if (isSignedIn)` reads as "if is signed in" whereas `if (!isSignedIn)` reads as "if not is signed in" which requires a bit more mental parsing.